app/components/primer/truncate.rb

```diff
@@ -6,7 +6,7 @@ class Truncate < Primer::Component
     status :beta
 
     DEFAULT_TAG = :div
-    TAG_OPTIONS = [DEFAULT_TAG, :span, :p].freeze
+    TAG_OPTIONS = [DEFAULT_TAG, :span, :p, :strong].freeze
```

Comment:
While we're here, do you think we should consider other similar text elements like, `<b>` and `<i>`?

Reply:
I was looking up the [phrasing content tags](https://html.spec.whatwg.org/#phrasing-content) and I feel there is a lot to add if we want to be exhaustive.
`<b>` and `<i>` for sure!

I feel `<em>`, `<samp>`, `<small>` and `<u>` can also be used with truncated text.

And `<bdi>` and `<bdo>` for bidirecitonal text?

I don't see anything bad about giving more semantic options to the user, but I don't know how we want to tackle that in the project in general?

Reply:
Good point. Let's leave it out for now :)